lib/awesome_print/formatter.rb

```diff
@@ -318,6 +318,26 @@ def plain_single_line
 
     # Utility methods.
     #------------------------------------------------------------------------------
+    def convert_to_hash(object)
+      if not object.respond_to?(:to_hash)
```

Comment:
Use the `!` over the `not` take a look at https://github.com/bbatsov/ruby-style-guide#bang-not-not